apis/python/tests/test_tiledbobject.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+import tiledbsoma as soma
+
+
+# Checking that objects _do_ exist is already done (thoroughly) in other
+# tests. Here we check the negative cases.
+#
+# Specifically, we're checking these don't raise, and all return False.
+
+
+@pytest.mark.parametrize(
+    "uri",
+    ["/nonesuch/no/nope/never/ever", "foo://bar", "s3://@@@@ILLEGAL@@@@"],
+)
+@pytest.mark.parametrize(
```

Comment:
the other negative test not covered is the check for a validate SOMA object of the expected type.

Ie.., soma.DataFrame.exists checks for the right TileDB metadata that matches the expected class type.

Reply:
good call @bkmartinjr , thanks for the code-review feedback